content/pt/docs/setup/_index.md

```diff
@@ -0,0 +1,40 @@
+---
+no_issue: true
+title: Instalação
+main_menu: true
+weight: 30
+content_type: concept
+---
+
+<!-- overview -->
+
+Essa seção lista as diferentes formas de instalar e executar o Kubernetes. Quando você realiza a instalação de um cluster Kubernetes, deve decidir o tipo de instalação baseado em critérios como facilidade de manutenção, segurança, controle, quantidade de recursos disponíveis e a experiência necessária para gerenciar e operar o cluster.
+
+Você pode criar um cluster Kubernetes em uma máquina local, em cloud, em um datacenter on-premises ou ainda escolher uma oferta de um cluster Kubernetes gerenciado pelo seu provedor de cloud computing.
```

Comment:
_computação em cloud_?

Reply:
There are some terms that are really ugly when translated to portuguese :) It's better known the term 'cloud computing' here that 'computação em nuvem' when you're referring to a provider.

So I've preferred to keep the original term instead of making the literal translation but maybe we can think about using some different formatting (italic, maybe) to separate technical terms, wdyt? Don't know if the Markdown support this :)

Reply:
Sure.
```markdown
You can use _italics_, eg:
cloud computing (_computação em nuvem_).
```

Reply:
Answering this, yes, italic formatting is supported so IMO we should follow with italic into untranslated technical terms, sounds good? :D

Reply:
Ricardo, can you translate cloud to "nuvem", because this term is very common in Portuguese. Therefore the better option to translate cloud computing is "computação em nuvem". What do you think?

Reply:
I can't say 'I like it' (sorry! haha), but no problems translating this also :) 'computação em nuvem' is a term that exists yes, so no problem with this!

Reply:
did as asked, ptal :) tks!